src/features/pages/home/live-pricing/market-tabs/utils.tsx

```diff
@@ -15,7 +17,7 @@ export const market_buttons: TMarketButtons[] = [
         src: Forex,
         selected_src: SelectedForex,
         button_text: 'Forex',
-        market_name: 'forex',
+        market_name: 'fx',
```

Comment:
why are we changing the names ? 🙈

Reply:
<img width="259" alt="Screenshot 2023-08-30 at 4 31 15 PM" src="https://github.com/binary-com/deriv-com/assets/121229483/3acd956e-af40-4e22-bd11-80829b9d2a56">

Hi @mitra-deriv because now in API response we are getting names in this formate